src/Uno.UI/Microsoft/UI/Xaml/Controls/TabView/TabViewItem.cs

```diff
@@ -135,6 +154,8 @@ private void OnIsSelectedPropertyChanged(DependencyObject sender, DependencyProp
 		private void UpdateShadow()
 		{
 			if (SharedHelpers.IsThemeShadowAvailable())
+			// TODO Uno: Can't access XamlControlsResources from Uno.UI
+			//&& !Microsoft.UI.Xaml.Controls.XamlControlsResources.IsUsingControlsResourcesVersion2)
```

Comment:
Once linking is available, we'll probably be able to merge fluent styles back to Uno.UI